From 52a9607818c903d83140f4b0643d0543875d47c6 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Thu, 7 Oct 2021 10:41:23 +0000
Subject: GCC: fix brace-initialization with NoDestructor

GCC requires brace-initialization for NoDestructor and std::vector<T>
for base_types. Therefore change to double brace-initialzation.

Bug: 819294
Change-Id: I24472f0607bd78917224d8b03de366274ba1d597
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3208021
Reviewed-by: Martin Robinson <mrobinson@igalia.com>
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Cr-Commit-Position: refs/heads/main@{#929124}
---
 ui/accessibility/platform/ax_platform_node_auralinux.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ui/accessibility/platform/ax_platform_node_auralinux.cc b/ui/accessibility/platform/ax_platform_node_auralinux.cc
index aff693dea500b..834c328d1d9b8 100644
--- a/ui/accessibility/platform/ax_platform_node_auralinux.cc
+++ b/ui/accessibility/platform/ax_platform_node_auralinux.cc
@@ -5136,8 +5136,8 @@ std::pair<int, int> AXPlatformNodeAuraLinux::GetSelectionOffsetsForAtk() {
 std::vector<ax::mojom::Action> AXPlatformNodeAuraLinux::GetSupportedActions()
     const {
   static const base::NoDestructor<std::vector<ax::mojom::Action>>
-      kActionsThatCanBeExposedViaAtkAction(
-          {ax::mojom::Action::kDecrement, ax::mojom::Action::kIncrement});
+      kActionsThatCanBeExposedViaAtkAction{
+          {ax::mojom::Action::kDecrement, ax::mojom::Action::kIncrement}};
 
   // The default action is always included and exposed at the first index.
   std::vector<ax::mojom::Action> supported_actions = {
-- 
2.30.2

